docs/source/user_guide/dataset_creation/datasets.rst

```diff
@@ -1798,12 +1832,31 @@ should implement is determined by the type of dataset that you are importing.
 
             Args:
                 dataset_dir: the dataset directory
-                *args: additional positional arguments for your importer
+                skip_unlabeled (False): whether to skip unlabeled images when importing
+                shuffle (False): whether to randomly shuffle the order in which the
+                    samples are imported
+                seed (None): a random seed to use when shuffling
+                max_samples (None): a maximum number of samples to import. By default,
+                    all samples are imported
                 **kwargs: additional keyword arguments for your importer
             """
 
-            def __init__(self, dataset_dir, *args, **kwargs):
-                super().__init__(dataset_dir)
+            def __init__(
+                self,
+                dataset_dir,
+                skip_unlabeled=False,
+                shuffle=False,
+                seed=None,
+                max_samples=None,
+                **kwargs,
```

Comment:
Ending argument lists with `,` is not supported in Python 3.5. I see 4 places where this was added in this PR, and that's why the Python 3.5 tests are failing.

I'd be open to dropping support for Python 3.5, since it reached end-of-life around 10 days ago, but that should be done in 0.6.0, not 0.5.x.

Reply:
grrr this again. I deleted the commas. No reason to drop Python 3.5 support for something easily fixable like this.